src/crimson/os/seastore/transaction.h

```diff
@@ -286,7 +287,12 @@ class Transaction {
     {
       auto where = read_set.find(placeholder.get_paddr());
       assert(where != read_set.end());
```

Comment:
Towards merging, can we add a debug log here instead?

Reply:
Yeah, will do.

Reply:
I've just turned this assert into an `unlikely if` and added logs for it, please take a look, thanks:-)